pkg/uninstall/uninstall.go

```diff
@@ -97,3 +135,35 @@ func (obj *uninstaller) createWatcher(ctx context.Context, pkg ctrlpkg.Package)
 		return nil, fmt.Errorf("unexpected object kind: %v", pkg.GroupVersionKind().Kind)
 	}
 }
+
+func (uninstaller *uninstaller) deleteNamespace(ctx context.Context, namespace string) {
+	clientset := cliutils.KubernetesClient(ctx)
+	err := clientset.CoreV1().Namespaces().Delete(ctx, namespace, metav1.DeleteOptions{})
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "❌ error deleting namespace: %v\n", err)
+		cliutils.ExitWithError()
+	}
```

Comment:
since this feature is going to used by the UI as well, the error should be returned instead so it can be handled downstream